packages/api-client/src/types/variable.types.d.ts

```diff
@@ -0,0 +1,58 @@
+export interface CreateVariableRequest {
+  projectId: string
+  name: string
+  entries: Entries[]
+}
+export interface Entries {
+  value: string
+  environmentId: string
+}
+export interface CreateVariableResponse {
+  id: string
+  name: string
+  createdAt: string
+  updatedAt: string
+  note: string | null
+  lastUpdatedById: string
+  projectId: string
+  project: Project
+  versions: Entries[]
+}
+
+export interface Project {
+  workspaceId: string
+}
+export interface UpdateVariableRequest {
+  variableId?: string
+  name?: string
+  entries?: Entries[]
+}
+export interface UpdateVariableResponse {
+  variable: Variable
+  updatedVersions: Entries[]
+}
+
+export interface Variable {
```

Comment:
Inline this

Reply:
You would need to move https://github.com/keyshade-xyz/keyshade/pull/395/files#diff-a5cc3a8d29899e24783200cd22d84f671e390b3fb0efc276f85840fc6d5ed0f3R41-R45 to https://github.com/keyshade-xyz/keyshade/pull/395/files#diff-a5cc3a8d29899e24783200cd22d84f671e390b3fb0efc276f85840fc6d5ed0f3R37